src/SmartComponents/DriftPage/DriftTable/ComparisonHeader/ComparisonHeader.js

```diff
@@ -47,6 +48,13 @@ class ComparisonHeader extends Component {
         setHistory();
     }
 
+    renderLoadingSystems() {
+        let row = [];
```

Comment:
I would suggest just simple one liner to achieve the same purpose:

`return [<td><Skeleton size={ SkeletonSize.md } /></td>]`